File: cpp_results/instruct_generations/cpp/files/question-53-pass-1.cpp
#include <bits/stdc++.h> 
 using namespace std ;
bool isPossible(int A, int B) {
    int prime = 2;
    while (prime <= A) {
        if (A % prime == 0 && B % prime == 0) {
            return true;
        }
        prime++;
    }
    return false;
}

File: results/fine_tuning/test_orig_files/question-53-pass-1.py
def isPossible(A, B):

def isPossible(A, B):
    if A == B:
        return True
    if A < B:
        return False

    for i in range(2, B+1):
        if A % i == 0 and B % i == 0:
            return True

    return False
